sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeGenerator.scala

```diff
@@ -1309,6 +1309,23 @@ object CodeGenerator extends Logging {
   // bytecode instruction
   final val MUTABLESTATEARRAY_SIZE_LIMIT = 32768
 
+  // The java source code generated by whole codegen is compiled on the Driver side and sent to
```

Comment:
Hi @beliefer ,

I'm really sorry to post edit suggestions after the PR is merged. I noticed this PR just now...

If we have a chance to make follow-up changes, could you please update `java code` to `Java code`, and `whole codegen` to `whole-stage codegen`?

Thanks!

Reply:
OK. Thanks for your remind.

Reply:
@beliefer can you send a followup PR to address it?

Reply:
@cloud-fan @rednaxelafx I make a followup PR https://github.com/apache/spark/pull/28112.

Reply:
This one? https://github.com/apache/spark/pull/28112

Reply:
Yes